redisson/src/test/java/org/redisson/RedissonShardedTopicTest.java

```diff
@@ -1,41 +1,58 @@
 package org.redisson;
```

Comment:
please rollback all changes to this file

Reply:
can I propose these changes in another PR?

Reply:
sure you can